src/rbdmap

```diff
@@ -17,8 +17,6 @@ do_map() {
 		  ""|\#*)
 			continue
 			;;
-		  */*)
```

Comment:
Isn't this for the case where the first column is "pool/image"? You don't want it to fall through to the next statement where "rbd" will be defaulted to the pool name.